src/errorHandler.ts

```diff
@@ -100,7 +114,7 @@ export function safeExecution(
  */
 export function setupErrorHandler(): Disposable {
     // Setup the statusBarItem
-    statusBarItem = window.createStatusBarItem();
+    statusBarItem = window.createStatusBarItem(StatusBarAlignment.Right, -1);
     statusBarItem.text = 'Prettier';
     statusBarItem.show();
```

Comment:
We should check if current active editor is in `allEnabledLanguages`. If I open VSCode on any file it will show up.